assets/src/amp-validation/counts/index.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { isPlainObject } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import domReady from '@wordpress/dom-ready';
+
+/**
+ * Internal dependencies
+ */
+import './style.css';
+
+/**
+ * Updates a menu item with its count.
+ *
+ * If the count is not a number or is `0`, the element that contains the count is instead removed (as it would be no longer relevant).
+ *
+ * @param {HTMLElement} itemEl Menu item element.
+ * @param {number} count Count to set.
+ */
+function updateMenuItem( itemEl, count ) {
+	count = Math.abs( count );
+
```

Comment:
I'm thinking `abs()` may be unnecessary here?

```suggestion
```

Reply:
In retrospect, yea not necessary. Removed in https://github.com/ampproject/amp-wp/pull/5900/commits/8d90f2d280b8fc3dced914e4f545e1e46182c909.